### ISSUE TITLE
Modify closed captions to omit words/ sentence

### ISSUE THREAD

**Issue by mchangrh** [ajayyy/SponsorBlock#1547]
mentioned in the [discord](https://discord.com/channels/603643120093233162/655247785561554945/1032114031114334219)

removing closed captions would allow for less jarring transitions and also help de-brand the video

**Comment by ajayyy** [comment]
Related to #538

**Comment by mchangrh** [comment]
> Related to #538

yes but moreso focusing on youtube closed captions rather than burned in subtitles

**Comment by erkinalp** [comment]
Yeah, #538 is related to open (baked-in) captions.
